quandl/model/dataset.py

```diff
@@ -29,6 +31,7 @@ def __init__(self, code, raw_data=None, **options):
         self.options = options
 
     def data(self, **options):
+        handle_not_found_error = options.pop('handle_not_found_error', False)
```

Comment:
Ahh. Maybe we can put in a comment about that. It was not exactly clear why we were doing that at first glance.